src/session.ts

```diff
@@ -43,30 +56,40 @@ export interface Store extends Readable<State> {
   connectWalletConnect(config: Config): Promise<void>;
   updateBalance(n: BigNumber): void;
   connectSeed(seed: { id: string; session: SeedSession }): void;
-  refreshBalance(config: Config): Promise<void>;
+  refreshBalance(): Promise<void>;
   setTxSigning(): void;
   setTxPending(tx: TransactionResponse): void;
   setTxConfirmed(tx: TransactionReceipt): void;
-  setChangedAccount(address: string, config: Config): void;
+  setChangedAccount(address: string, signer: Signer): void;
 }
 
+// This is a buffer for the Config class to be available in this file, without making it a store just yet.
+// TODO: We should be thinking on making config a store, similar to session, to be more accesible.
+let sessionConfig: Config | null = null;
```

Comment:
I accept this to be a not so nice workaround, but there are places like in the `window.ethereum` event handlers, were I really wanted to have a current `Config` instance.

This shows IMO that we should eventually move `Config` to be a store similar to `session` to allow us to be more easily accesible..